apps/prs/src/app/login/login.component.scss

```diff
@@ -0,0 +1,6 @@
+:host {
```

Comment:
Nit: This extra layer could be removed or does it exist to increase specificity? (if so maybe a comment?)